CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 0.5.3
+## 0.5.2-dev2
 
 * Configure fallback for OCR token if paddleocr doesn't work to use tesseract
 
```

Comment:
The pattern is to keep all changes consolidated until we do a release. So this section should be removed, the change should be listed in the `0.5.2-dev1` section, and then the `0.5.2-dev1` header should be bumped to `0.5.2-dev2`.